fuzzers/structure_aware/baby_fuzzer_multi/src/main.rs

```diff
@@ -144,11 +144,7 @@ pub fn main() {
 
     // a generator here is not generalisable
     let initial = MultipartInput::from(
-        iter::repeat((
-            "part".to_string(),
-            BytesInput::new(vec![b'h', b'e', b'l', b'l', b'o']),
-        ))
-        .take(4),
+        iter::repeat(("part".to_string(), BytesInput::new(b"hello".to_vec()))).take(4),
```

Comment:
BytesInput implements from &[u8], shouldn't just "into" work?

Reply:
Yeah, it's a `&[u8; 5]`, and for some reason, that isn't automatically converted to a `&[u8]`, which is expected by the `From` impl.